src/Symfony/Bundle/WebProfilerBundle/Profiler/TemplateManager.php

```diff
@@ -134,4 +136,49 @@ protected function templateExists($template)
 
         return false;
     }
+
+    /**
+     * It changes the default order of collector templates to show them in a
+     * different order which is better for design and aesthetic reasons.
+     *
+     * @param array $templates
+     *
+     * @return array
+     */
+    private function reorderTemplates($templates)
```

Comment:
I've just made this change ... which has allowed us to remove a lot of lines of code. \o/  Thanks!